src/client/http_handler.rs

```diff
@@ -32,20 +41,53 @@ impl HttpVerb {
 pub struct HttpResponse {
     content: String,
     status: u16,
-    endpoint: Endpoint,
 }
 
 impl HttpResponse {
-    pub fn new(content: String, status: u16, endpoint: Endpoint) -> Self {
+    pub fn new(content: String, status: u16) -> Self {
+        Self { content, status }
+    }
+
+    pub fn content(&self) -> &str {
+        &self.content
+    }
+
+    pub fn status(&self) -> u16 {
+        self.status
+    }
+
+    // pub fn endpoint(&self) -> Endpoint {
```

Comment:
What was this used by that doesn't need it anymore?

Reply:
The methods that now return a HttpStreamingResponse used to use this. Now HttpResponse is only used by the Capella methods which never populated/read this field.